resources/proxytrack.rb

```diff
@@ -0,0 +1,43 @@
+resource_name :proxytrack
+
+default_action :create
+
+property :proxy_address,      String
+property :proxy_port,         Integer
+property :icp_address,        String, default: 'localhost'
+property :icp_port,           Integer
+property :httrack_file_paths, Array
+
+action :create do
+  run_context.include_recipe 'proxytrack::default'
+
+  proxy_cmd = "proxytrack #{new_resource.proxy_address}:#{new_resource.proxy_port} #{new_resource.icp_address}:#{new_resource.icp_port} #{new_resource.httrack_file_paths.join(' ')}"
+
+  grep_proc_cmd = "ps aux | grep \"#{proxy_cmd}$\""
+
+  execute 'Launch proxytrack proxy' do
+    command "if [[ ! $(#{grep_proc_cmd}) ]]; then #{proxy_cmd} & fi"
+  end
```

Comment:
Please use the [systemd_unit resource](https://docs.chef.io/resource_systemd_unit.html) to manage this instead.